packages/typespec-ts/package.json

```diff
@@ -67,10 +67,10 @@
     "@azure-tools/cadl-ranch-specs": "^0.38.0",
```

Comment:
We have issue with the latest cadl-ranch and will work on fix and cadl-ranch upgrading in another pr.